frontend/packages/kubevirt-plugin/src/statuses/vm/vm-status.ts

```diff
@@ -309,6 +319,7 @@ export const getVMStatus = ({
     isV2VVMImportConversion(vm, vmImports) ||
     isBeingMigrated(vm, vmi, migrations) ||
     isBeingImported(vm, pods, pvcs, dataVolumes) ||
+    isDeleting(vm, vmi) ||
     isVMError(vm) ||
```

Comment:
IMO we should move it behind isVMError

Reply:
That is the bug, users see "vmi-error" while they need to see "deleting"

Reply:
this is not a vmi error but possible vm error - which should have a prefference IMO, as it could say a reason why something cannot be deleted even though it has deleteionTimestamp

Reply:
ok, moved